$Id$

	bugfix from MirBSD ntpd: do not hardcode the number
	of required adjustment offsets, use the constant
	instead, MirOS commitid 10047041D2145A88E16

--- openntpd-3.9p1.orig/client.c	Sun May 14 05:28:58 2006
+++ openntpd-3.9p1/client.c	Mon Oct  8 12:08:30 2007
@@ -314,7 +314,7 @@ client_update(struct ntp_peer *p)
 				best = i;
 		}
 
-	if (good < 8)
+	if (good < OFFSET_ARRAY_SIZE)
 		return (-1);
 
 	memcpy(&p->update, &p->reply[best], sizeof(p->update));
